src/components/forDocs/DownloadButton.jsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Shows a button inside an anchor tag for downloading a particular SDK
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+
+export default function DownloadButton({ link, text, classes }) {
+  return (
+    <a href={link} target="_blank" rel="noreferrer">
+      <button
+        className={`px-4 py-2 rounded-md text-sm bg-blue-600 text-white hover:shadow-md hover:cursor-pointer border-none ${
```

Comment:
IMO this is hard to read

Reply:
How about adding classes util?

Reply:
We can import it here, but I don't see the value in doing so right now since there aren't many places that need it. What do you think?

Reply:
Makes sense

Reply:
Ack, resolving this. Will consider it if components start increasing and we run into this issue